src/main/java/mate/service/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package mate.service;
+
+import java.util.Optional;
+import mate.controller.exception.AuthenticationException;
+import mate.lib.Injector;
+import mate.lib.Service;
+import mate.model.Driver;
+
+@Service
+public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService = (DriverService) injector
+            .getInstance(DriverService.class);
```

Comment:
you can use DriverDao directly from this service

Reply:
Ok, I understand. But is it really metter?